src/app/components/calendar/calendar.ts

```diff
@@ -1156,12 +1156,56 @@ export class Calendar implements OnInit,OnDestroy,ControlValueAccessor {
         this.onYearChange.emit({ month: this.currentMonth + 1, year: this.currentYear });
         this.createMonths(this.currentMonth, this.currentYear);
     }
+
+    validateTime(hour, minute, second): boolean {
+        let valid: boolean = true;
+        let value = this.value;
+        if (this.isRangeSelection()) {
+            value = this.value[1] || this.value[0];
+        }
+        if (this.isMultipleSelection()) {
+            value = this.value[this.value.length - 1];
+        }
+        let valueDateString = value ? value.toDateString() : null;
+
+		if (this.minDate && valueDateString && this.minDate.toDateString() === valueDateString) {
+			if (this.minDate.getHours() > hour) {
```

Comment:
This doesn't  take into account if the hour is in 12 format.  The minDate and maxDate are in 24 hour format.

I am using both date and time selection.